dataAccessObjects.py

```diff
@@ -360,6 +360,14 @@ def getCwbImgIds(self):
         Ids = self.db.query(sql)
         return Ids
 
+    def getDisplayImgs(self,userId=None):
+        sql = "SELECT img_id, img_upload_time, img_file_name, img_start_time, img_end_time, img_start_date, img_end_date, type_id, img_thumbnail_name, img_display_time, img_display_count " \
+                + "FROM image_data WHERE img_is_delete=0 "
+        if userId:
+            sql += " AND user_id={userId}".format(userId=userId)
+        ret = self.db.query(sql)
+        return ret
```

Comment:
No, if ret is `None` then it won't enter the for-loop in display_api.py at line 53~55